### PR TITLE
Added section Workflow

### PR DESCRIPTION
**Pull request by Asawari-P**

### CODE REVIEW

File: content/workflow.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
+---
+id: workflow
+title: Workflow
+sidebar_label: Workflow
+slug:
```
Comment (zcourts) [inline content/workflow.md:5]: You need to have a slug. The slug is the URL when the documentation is published.
`slug: /workflow`

File: content/workflow.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+| execAs           | String       | An ArcQL query to find the account e.g. hypi.id = 'user123' to find by id or username = 'blah' to find by username.<br><br>If present, execution of the steps in the Workflow will be done as this account. If not specified, it defaults to the account making the request                               |
+| Async            | Boolean      | Set to \`true\` for asynchronous processing                                                                                                                                                                                                                                                               |
+| Parallel         | Boolean      | If present AND true, all steps in the workflow are executed at the same time.                                                                                                                                                                                                                             |
```
Comment (zcourts) [inline content/workflow.md:33]: `parallel` - the uppercase matters

File: content/workflow.md [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+| maxExecutionTime | String       | Specifies the max time an async task should be allowed to execute. When this time has elapsed the task will be killed.<br><br>The format is ISO8601 durations [https://en.wikipedia.org/wiki/ISO_8601](https://en.wikipedia.org/wiki/ISO_8601)<br><br>#Durations e.g. P1M is 1 month and PT1M is 1 minute |
+| Steps            | WorkflowStep | Holds details of each workflow step                                                                                                                                                                                                                                                                       |
```
Comment (zcourts) [inline content/workflow.md:35]: `steps` - case matters

File: content/workflow.md [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+    name: String
+    fn: GraphQLRef!
+    order: Int!
+}
+```
+
+| Parameter | Type       | Description                                                                                                                                                                                                  |
+|-----------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Name      | String     | The name of the step                                                                                                                                                                                         |
+| Fn        | GraphQLRef | The function to execute for this step. The data returned by the step can subsequently be used in other steps. Set the type of the functions as query, mutation, or Subscription. Set the field to step name. |
```
Comment (zcourts) [inline content/workflow.md:49]: `fn`

File: content/workflow.md [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+| Order     | Int        | Order of the step                                                                                                                                                                                            |
```
Comment (zcourts) [inline content/workflow.md:50]: `order`

File: content/workflow.md [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+            }
+        }\"""
+    )
+""")
+}
+```
+
+> Important notes to understand the schema!
+
+The first step can have any parameters you want. For the other steps, some rules define how the system maps parameters to the function in these steps.
```
Comment (zcourts) [inline content/workflow.md:187]: `rules define` should be `rules defined`

File: content/workflow.md [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+                 }
+                 }
+             ]
+             }
+         ]
+     }
+}
+```
+
+To use this workflow execute it like any other workflow.
```
Comment (zcourts) [inline content/workflow.md:277]: `To use this workflow execute it like any other function.`

**Review by zcourts — APPROVED** [review]

File: content/workflow.md (hunk 1 above)
Comment (Asawari-P) [reply content/workflow.md:5]: slug : workflow ? or entire path?

File: content/workflow.md (hunk 2 above)
Comment (Asawari-P) [reply content/workflow.md:33]: The first letters in the table were auto corrected with capital letters. Will change

File: content/workflow.md (hunk 6 above)
Comment (Asawari-P) [reply content/workflow.md:187]: are defined?

File: content/workflow.md [hunk 8]
```diff
@@ -0,0 +1,406 @@
+---
+id: workflow
+title: Workflow
+sidebar_label: Workflow
+slug: https://docs.hypi.io/docs/workflow
```
Comment (zcourts) [inline content/workflow.md:5]: The example I gave was exactly what you needed. Don't put the http..., just `slug: /workflow` the documentation framework will turn this into `https://docs.hypi.io/docs/workflow`

Comment (Asawari-P) [reply content/workflow.md]: corrected

**Pull request merged**